hq/README.md

```diff
@@ -56,6 +56,9 @@ AWS_ACCOUNTS=[
 ]
 ```
 
+### AWS Security Policies
+See ```watched-account``` cloud formation template for the security policies needed to run security-hq.
```

Comment:
Minor (don't let me prevent merge): 
- those triple quotes could be single ones
- we have been treating cloudformation as one word 😄